node_lambnik/src/demo/pwd_inlets_index.html

```diff
@@ -35,39 +62,71 @@ <h3>PWD Inlets</h3>
     document.addEventListener('DOMContentLoaded', function onload() {
         // create map object
         var map = L.map('map', {
-            center: [39.96, -75.15], zoom: 13
+            center: [39.96, -75.15], zoom: 12
         });
 
-        // add basemap
-        L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
-            maxZoom: 19,
-            attribution: '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>'
-        }).addTo(map);
-
-
         // add tileserver
         // un-comment out the path that you want to test
 
-        // var TILESERVER_PATH = 'https://d28f32z0lbuk2f.cloudfront.net/';
+        //var TILESERVER_PATH = 'https://d28f32z0lbuk2f.cloudfront.net/';
         var TILESERVER_PATH = 'http://localhost:9001/';
 
-        // add custom tile layer
-        L.tileLayer(TILESERVER_PATH + '/tile/{z}/{x}/{y}.png').addTo(map);
-
-        // add utf grid
-        var grid = new L.UtfGrid(TILESERVER_PATH + "/grid/{z}/{x}/{y}?utfFields=inlettype,owner", {
-            useJsonP: false
-        });
+        // Create/add tile layers
+        var layerKeys = {};
+        function createLayer(ownerName) {
+            lyr = L.tileLayer(TILESERVER_PATH + '/tile/{z}/{x}/{y}.png?layers=' + ownerName);
+            grd = new L.UtfGrid(TILESERVER_PATH + "/grid/{z}/{x}/{y}?utfFields=inlettype&layers=" + ownerName, {
+                useJsonP: false
+            });
+            grd.on('click', function(e) {
+                if (e.data) {
+                    alert("This inlet is of type " + e.data.inlettype);
+                }
+            });
 
-        map.addLayer(grid);
-        grid.on('click', function(e) {
-            if (e.data) {
-                alert("This inlet is owned by: " + e.data.owner + " and has type " + e.data.inlettype);
-                console.log(JSON.stringify(e.data))
+            layerKeys[ownerName] = {
+                layer: lyr,
+                grid: grd,
             }
+        }
+
+        createLayer('PWD');
+        createLayer('STATE');
+        createLayer('PARK');
+        createLayer('AIRPRT');
+        createLayer('LM');
+        createLayer('FEDERAL');
+        createLayer('PRIV');
```

Comment:
It's just code on a demo page, but the repetition here and the indirectness above are bothering me a little.  The latter especially, since I would expect `createLayer` to return the layer rather than modify an object from outside its scope.

I would vote for changing the function to return the layer and changing the object creation to:
```javascript
var layerKeys = ['PWD', 'STATE', 'PARK', 'AIRPRT', 'LM', 'FEDERAL', 'PRIV']
const layers = Object.assign(...layerKeys.map(key => ({[key]: createLayer(key)})));
```
(and using `layers` below rathar than `layerKeys`. more accurately descriptive)
That's an annoying number of closing parens with a curly brace thrown in for extra unreadability, but it has the advantage that it makes the object directly rather than as a side-effect.  And it's short.